training/training_job.js

```diff
@@ -157,6 +125,14 @@ async function taskReloadingExact(job) {
 }
 
 async function taskGenerateTrainingSet(job) {
+    await delay(0);
```

Comment:
can you add a comment why a `await delay(0)`  is necessary?

Reply:
I have to admit, I don't actually remember why this is necessary. It shouldn't matter, I'll remove it.